third_party/src/default/assets/js/main.js

```diff
@@ -428,17 +428,17 @@ function renderSimpleHTMLRecursive(obj, package, spacing) {
                 shownPackages.push(package);
             }
             var href = '';
-            if (window.location.href.indexOf('modules') == -1) {
+            if (window.location.href.indexOf('/modules/') == -1) {
                 href = 'modules/';
             }
-            if (window.location.href.indexOf('assets') > -1 ||
-                window.location.href.indexOf('classes') > -1 ||
-                window.location.href.indexOf('enums') > -1 ||
-                window.location.href.indexOf('interfaces') > -1) {
+            if (window.location.href.indexOf('/assets/') > -1 ||
+                window.location.href.indexOf('/classes/') > -1 ||
+                window.location.href.indexOf('/enums/') > -1 ||
+                window.location.href.indexOf('/interfaces/') > -1) {
                 href = '../modules/';
             }
-            if (window.location.href.indexOf('modules') > -1) {
-                href = '../modules/' + href;
+            if (window.location.href.indexOf('/modules/') > -1) {
```

Comment:
@Fleker Please take care of this part.
For href "127.0.0.1/docs/modules/****", the original code will make the link to be "127.0.0.1/docs/modules/modules/****", which is not correct

Reply:
This change would also need to go into `NavigationAccordion.ts`.